build/openresty/patches/nginx-1.21.4_06-dynamic_log_level.patch

```diff
@@ -0,0 +1,211 @@
+diff --git a/bundle/nginx-1.21.4/src/core/ngx_log.c b/bundle/nginx-1.21.4/src/core/ngx_log.c
+index eb7a989..0862d4d 100644
+--- a/bundle/nginx-1.21.4/src/core/ngx_log.c
++++ b/bundle/nginx-1.21.4/src/core/ngx_log.c
+@@ -171,8 +171,12 @@ ngx_log_error_core(ngx_uint_t level, ngx_log_t *log, ngx_err_t err,
+     debug_connection = (log->log_level & NGX_LOG_DEBUG_CONNECTION) != 0;
+
+     while (log) {
+-
++#if (NGX_HTTP_LUA_KONG)
++        if (ngx_http_lua_kong_get_dynamic_log_level(log->log_level) < level &&
```

Comment:
As we talked offline, this is a patch for the core module, but here we invoke a *http* function, it makes the code a little weird.